flash

```diff
@@ -213,7 +213,7 @@ while [[ -z $PREVIEWER_PARAMTERS ]]; do
   case $PREVIEWER in
```

Comment:
double quote the variable unless there's a specific reason not to: `"$PREVIEWER"`